### PR TITLE
Classifer: handle default frame for multi-image subjects

### PR DESCRIPTION
**Pull request by goplayoutside3**
## Package
lib-classifier

## Linked Issue and/or Talk Post
https://github.com/zooniverse/front-end-monorepo/issues/3792

## Describe your changes
For subjects with multiple images in the `locations` array, SubjectViewerStore's `resetSubject()` (called upon subject load) handles `metadata.default_frame`. This PR specifies that a default frame of 2 should be set as index 1, or a default frame of 1 should be set as index 0, etc.

## How to Review

Flipbook's storybook now has a case where the subject's default frame is 3. Viewing that story on this branch should correctly show frame 3 selected when the subject loads.

As mentioned in Issue #3792, Correct-a-cell's subjects have `default_frame` of 3, so viewing the project [locally](https://local.zooniverse.org:3000/projects/h-spiers/etch-a-cell/classify/workflow/8921?env=production) should correctly show frame 3 selected when the subject loads.

WIldwatch Burrowing Owl's subjects do not specify a default frame, so viewing the project [locally](https://local.zooniverse.org:3000/projects/sandiegozooglobal/wildwatch-burrowing-owl/classify/workflow/21141?env=production) should correctly show frame 1 selected when the subject loads.

# Checklist
_PR Creator - Please cater the checklist to fit the review needed for your code changes._
_PR Reviewer - Use the checklist during your review. Each point should be checkmarked or discussed before PR approval._

## General
- [ ] Tests are passing locally and on Github
- [ ] Documentation is up to date and changelog has been updated if appropriate
- [ ] You can `yarn panic && yarn bootstrap` or `docker-compose up --build` and FEM works as expected
- [ ] FEM works in all major desktop browsers: Firefox, Chrome, Edge, Safari (Use Browserstack account as needed)
- [ ] FEM works in a mobile browser

## General UX
Example Staging Project: [i-fancy-cats](https://local.zooniverse.org:3000/projects/brooke/i-fancy-cats)
- [ ] All pages of a FEM project load: Home Page, Classify Page, and About Pages
- [ ] Can submit a classification
- [ ] Can sign-in and sign-out
- [ ] The component is accessible
  - Can be used with a screen reader [BBC guide to VoiceOver](https://bbc.github.io/accessibility-news-and-you/assistive-technology/testing-steps/voiceover-mac.html)
  - Can be used from the keyboard [WebAIM guide to keyboard testing](https://webaim.org/techniques/keyboard/#testing)
  - It is passing accessibility checks in the storybook

## Bug Fix
- [ ] The PR creator has listed user actions to use when testing if bug is fixed
- [ ] The bug is fixed
- [ ] Unit tests are added or updated

### CODE REVIEW

**Comment by mcbouslog** [comment]
Apologies for what might be a brief possibly irrelevant comment, but I don't want to forget something that just came up in a meeting, I'm not sure if/how frame is saved in annotations (I'm thinking transcription annotations) and if this will change how/if frame is saved in annotations. Want to avoid if the same annotation before this PR will have a different frame after this PR.

**Comment by goplayoutside3** [comment]
Good point! I don't think this will have an effect on any live FEM projects with multi-image subjects, but I'll test them out with this branch, and double-check that those projects don't have a default frame specified via the project builder.

**Comment by eatyourgreens** [comment]
The frame index in annotations would be covered by tests, right? So changing it here would fail those tests. 

FWIW I don’t see anything in the code changes here that would change classifications.

File: packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/FlipbookViewer/FlipbookViewer.stories.js [hunk 1]
```diff
@@ -34,10 +34,38 @@ const mockSubject = SubjectFactory.build({
...
+  metadata: {
+    default_frame: 3
+  }
+})
+
 const store = mockStore({
   subject: mockSubject
 })
 
+const store2 = mockStore({
```
Comment (eatyourgreens) [inline packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/FlipbookViewer/FlipbookViewer.stories.js:65]: Is there a better name for this store?

Comment (goplayoutside3) [reply packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/FlipbookViewer/FlipbookViewer.stories.js]: I renamed it to `storeWithDefaultFrame`

**Comment by goplayoutside3** [comment]
> FWIW I don’t see anything in the code changes here that would change classifications.

Agreed, I don't see any change to live FEM transcription projects with multiple images in a subject.

**Review by eatyourgreens — APPROVED** [review]
Code read looks good. I’ll check the stories tomorrow but I’m happy for this to be merged as-is. 

EDIT: Storybook looks good. The subject's default frame could maybe be added to the story controls.

**Pull request merged**